src/__tests__/aggregatorExecuteLooksRareV2.test.ts

```diff
@@ -45,8 +45,8 @@ describe("LooksRareAggregator class", () => {
     const makerOrder: Maker = {
       quoteType: QuoteType.Ask,
       globalNonce: 0,
-      subsetNonce: 0,
-      orderNonce: 0,
+      subsetNonce: nonce || 0,
+      orderNonce: nonce || 0,
```

Comment:
Can you remind me whether you can set a default value for `nonce` in the function argument definition?

Reply:
Yes, it works and I fixed it!